src/eckit/config/LocalConfiguration.h

```diff
@@ -76,6 +77,18 @@ class LocalConfiguration : public Configuration, public Configured {
         return set(name, abstract_pointers.data(), abstract_pointers.size());
     }
 
+    template <typename ConfigurationT,
+              typename = std::enable_if_t<std::is_base_of_v<Configuration, std::decay_t<ConfigurationT>>>>
+    LocalConfiguration& set(const std::string& name, std::initializer_list<ConfigurationT>&& value) {
+        std::vector<const Configuration*> abstract_pointers;
+        abstract_pointers.reserve(value.size());
+        for (auto& v: value) {
+            abstract_pointers.emplace_back(&v);
+        }
+        return set(name, abstract_pointers.data(), abstract_pointers.size());
+    }
```

Comment:
This change was required to support a failing fdb5 test.
e.g. https://github.com/ecmwf/eckit/actions/runs/7641531854/job/20819349927#step:3:2397